container-build-task/container-build-helper.ts

```diff
@@ -43,6 +43,21 @@ export function buildFullImageTag(
 }
 
 /**
+ * Adds listeners to the gcloud ToolRunner that output stdout and stderr to appropriate console logging levels.
+ * @param gcloud The gcloud tool runner.
+ */
+export function listenToOutput(gcloud: ToolRunner) {
+  gcloud.on(
+    'stdline', (line: string) => {
+      task.debug(line);
+    });
+  gcloud.on(
+    'errline', (line: string) => {
+      console.log(line);
+    });
+}
+
+/**
  * Adds a listener to the gcloud ToolRunner that log stdout until the  start of
  * the JSON output. It will then collect but not log the JSON output, parse it,
```

Comment:
Is this still an issue? `stdout` should only contain the valid output you requested now a days. If there are still cases when you receive invalid output when using `--format=json` we should report those to `gcloud`.

Reply:
I have a bug open with them, but it hasn't gotten attention in a while.